### PR TITLE
Fix scale request when capacity in Paused state

### PR DESCRIPTION
**Pull request by selasagi**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X ] Please add REST spec PR link to the SDK PR
specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/2017-10-01/powerbidedicated.json
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
1. Please link the related REST spec
2. Create a generate.ps1 similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code
3. Bump the version number to `0.10.0-preview` and update the `PackageReleaseNotes`

**Comment by selasagi** [comment]
Spec PR link - 
https://github.com/Azure/azure-rest-api-specs/pull/1905

File: src/SDKs/PowerBIDedicated/generate.ps1 [hunk 1]
```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "PowerBIDedicated/resource-manager" -AutoRestVersion "latest"
```
Comment (dsgouda) [inline src/SDKs/PowerBIDedicated/generate.ps1]: Move this to under `Management.PowerBIDedicated` dir

File: src/SDKs/_metadata/powerbidedicated_resource-manager.txt [hunk 2]
```diff
@@ -1,11 +1,20 @@
-2017-11-08 22:41:50 UTC
-
+﻿Installing AutoRest version: latest
```
Comment (dsgouda) [inline src/SDKs/_metadata/powerbidedicated_resource-manager.txt]: A file named SDKInfo_*.cs should be created with the rest of the SDK. Commit this file.
Also run `msbuild build.proj /t:Build /p:Scope=SDKs\PowerBIDedicated` and commit any changes generated

**Review by dsgouda — CHANGES_REQUESTED** [review]
Needs a few more changes

File: src/SDKs/_metadata/powerbidedicated_resource-manager.txt (hunk 2 above)
Comment (selasagi) [reply src/SDKs/_metadata/powerbidedicated_resource-manager.txt]: SDKInfo_*.cs does not created after run .\generate.ps1 and also after the build..

Comment (selasagi) [reply src/SDKs/_metadata/powerbidedicated_resource-manager.txt]: created. thanks

**Comment by selasagi** [comment]
Please see my response. Thanks

File: src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Microsoft.Azure.Management.PowerBIDedicated.csproj [hunk 3]
```diff
@@ -7,12 +7,13 @@
 		<PackageId>Microsoft.Azure.Management.PowerBIDedicated</PackageId>
 		<Description>Provides management capabilities for Azure PowerBI Dedicated accounts.</Description>
 		<AssemblyTitle>Microsoft Azure PowerBI Dedicated Management Library</AssemblyTitle>
-		<Version>0.9.0-preview</Version>
+		<Version>0.10.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Microsoft.Azure.Management.PowerBIDedicated.csproj]: Please bump the `AssemblyFileVersion` here to `0.10.0.0`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Run `msbuild build.proj /t:build /p:Scope=SDKs\PowerBIDedicated` and commit the changes to .props file [here ](https://github.com/selasagi/azure-sdk-for-net/blob/59c0735cc7bb46899a1fcd91743cb87f6978e80e/src/SDKs/PowerBIDedicated/AzSdk.RP.props)

**Pull request closed without merging**